vpr/src/timing/slack_evaluation.cpp

```diff
@@ -19,7 +31,24 @@ SetupSlackCrit::SetupSlackCrit(const AtomNetlist& netlist, const AtomLookup& net
     , netlist_lookup_(netlist_lookup)
     , pin_slacks_(netlist_.pins().size(), NAN)
     , pin_criticalities_(netlist_.pins().size(), NAN) {
-    //pass
+#if !defined(INCR_SLACK_UPDATE) || !defined(INCR_UPDATE_CRIT)
```

Comment:
Again, but shouldn't these be runtime modifications, not compile time?  How often is this constructor used?

Reply:
Done